src/cf-layout/src/cf-layout.less

```diff
@@ -665,18 +665,22 @@
     &_heading {
         .heading-1();
 
-        .respond-to-max(800px, {
+        .respond-to-max(@bp-sm-max, {
             .heading-2();
         });
     }
 
-    &_subhead {
+    &_subhead,
+    &_cta {
```

Comment:
The spec says `lead-paragraph`, should we remove the heading mixin and add that class to a markup example? 

> Subheading: Lead paragraph (Avenir Next Regular, 22px/28px), begins 15px below headline, 185 maximum character count (including spaces)